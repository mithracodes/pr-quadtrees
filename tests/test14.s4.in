144.9375 -37.8750 145.0000 -37.6875
