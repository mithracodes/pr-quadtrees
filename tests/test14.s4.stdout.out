144.9375 -37.8750 145.0000 -37.6875 --> SW NE SW NW NW NW NE NE SE NE SW NE NE SE SE NW NW NE SW NE NW NE SW SW SW NW SE NE NE NW NE SE SE SE SW SE NE SW SW NW NW NE NE SW SW SW SW NW SE NW NE SW SE SE NE NW NW NW NE SW SE SE NW NE SW NE NE SW SW SE SW NE SE NW NE SE SW SW SW SE NW SW SW SE NW NW SE NW SW NW NE SE NE NW SW NW SW NW SE SW NE SE NE NW NW SW SW NE SW SE SE NE SW NW SW SE NW SW SW NW NW SW SE SW NE SW NW NE SW SW NE SE NW SE SE NE NE SE SW SE NW SW NE SE NE SE NW NE NE SW SE SW NW NE NW SW NE SW SE NE NE SW NE NW NE SE NW NW SW NW NW NE SE SE SW SW NE NW NW NE NE NW SW SW SE NE SW NE SW SW SW SE NW NW SE NE SW SW SE SE NE SW SW SE SE NE SW NW NW SE NE NW SE SE NE NE SE SW NE NW SE SE SW SW SW NW SE SW NE SE SE SW SE NE SE SE NW SE SW NE NE SW SW NW NE NW NW SE NE NW SE SE NW NE NW SW NW NW SW NW SE NE SW NW NW NE NE NE NW SE NE NW NE NW SW SW SW SE SW NE SW NW NE NE NE SE NW SE SW NE SE SW NE SE SE NW NE SE SE NE SW NW SE NE SE SW NW NE SE NW NE SE NE NE NE SE SW NW SE NE NW NW SW NW NE SE SW NE SE NW SW NW NW NE SW NE SW SW SW NW SE NW NW SW SE NE SW NE SE NW NE SE SW SW NW NE SE NW SE NE NE SE SW SE NW SW NE SE NE SW SW SE SE NE SE NE NW SW SE NE SW SE SE SW NE SE NW SW SW NW SE NE NE SW SW SE SW SW SE SW NW SW SW NE SW NE NW SW SW NE NW SE NW NE NW NW SW SE NW NE SW NE NW SW SE NE SE SW NE NW NE NE SE SW NW SE SW NW NW NE NE SW SW NW NE SW SW SE NW NE NE SW SW NE SW SE NW SW NE NE SE NW SW NW NE SE NE NW SW SE NE SW SE SE NW SW NW SE NW NW NE SE SW NW NE NW NE NE SE SE SW NE SE SE NE NE SW SE SE SW SE SE SW NE SE NE NW NW NW NW NW NW NW NW NE SE NW NW NE NE SE NW SW SE SW SE SE NW NE SW SW NW SW SE NW NE NE NE SW SW NW SE SW NE NE SW NW SW SW SW SE NW NE NE NE NW NE NW SE NE NW NE NW NW NE SW NE SE NE NW SW NW SW NW SW SW SE NE SW SE SW SW SW NW NE SW SE NW SE SW SE NW SE NW SE NE NW NE NE SW SE SE NE NE NW SE SW NW SW SE NW NW NE NW SE SE SW NE SW NE SW NE NW SE SE NW NW NW SE SE NE NW SW NW NW NE NE SW NW SE SE SE NW SW SW SW SW NW NE SE SE SW NE SW SE NE NE SE NW SE SE NW SE NE NE NE SW NE SW SE NW SE SE SE NW NW NW NW SW SW NW SE SE SE NW NE NW SE NE NE NW NW SE NE NE NW SE SW NE SW SW SW NE NE SE NE SE NE NW NW SE NW NW SW SE SE NE NE NW SW NW NE NW NE NW NE SE SE SW NE NW NW NE SE NW SW SW NE SW NE SE SW NE NW NE NW SE NW SW NW SE NW SE SE NE NW NE SE NW SW NE SW SE SE SW NW SW SE NE NW NW NE SE NE NE SW SE SE SW NW NE SW NW NE SE NE SE NW SW SE NW NE NW NE SW SW NE SE NE SW NE NE NE SW NE NW NW NE NE NW NE SE NW NW NE NW SW NE SW NE SW SE SW NE NW NE SW NE SE SW NW SE SE NE SW NW SW NE SW NE NW NW SE NW SE SE SW NW NW SE NW SE SE SE SW SE NW NW NE SE NE SW NW NW SW NW SE NW NE SE NW NE SE NE SW NW NW SW SE SE NW SE SE NW NE NW NE SE NE SW NE SE SW NW NE NW NE NW NE NE NW NE NE SW NW SE SE SW SW NE SW NE NE SE SW SE SE NW SE NE SE NE SE SW SW SW NE SE SW NW NE NE SW SW NW NE SE NE NW NE SW SE SE SW SW NW SE NW NE NW SW NW SW NE SE NW SW NW SE NW NE NE SW SW SW NW NE NW NE SE SE SW NW SW NE NE NE NW NW SE SW NE NE SE SE NW SW SW NW NW NW SW SE NW NW NE SE NE NW NE SE NW NE SW SE NW NE SE NE SE NW SE NW SW SE NE NE SW SE NW NW NW NW NE SE NE SW NE SE NW SE SW NE SE SW NW SW NW NE SE NW SW SE NE SW SW SE NE NW NE SE SW NW SE NW NE SW NW NW NW SW NE NE SW NW NE SW NE SE NE NW NE SE NE NE SW SW NE SW NE NE SE SW NW SW SW NW SE NE NW NE SE NW NE SW SE SE SW SE NW SW SE SW SE NW NE SE NE SW SE NW SE SW NE SE SW NW SW SE SE SW SW SW SW SE SE NW NE SE NW SE SW SW SE NE SE NE SW SW SW NE SW NW SE SW NW SW SE SE NW NW SW SW NE NE NW NE SE SW SW NW NE SE NW SW NE SW SE SW SE SE NW SE SW NE SW NE NW NE SE SE SW NW NW NW SW NE NE NE SE NW SW SW SE SE SW SE NW SW NE SE SE NW SE SE SW SW NE SW NE NE SE SW SW SE SE SW SW SW NW NW NW SE SE NE SE SE NE NE NW SE SW SW SW NE NE NW NE SW NE SE NW SW SE NW NE NE SE NE NE NE NE NW SW SE SE NE SW NW NE NW SW NW NE NE NE NW SW NW SE SW SW NW NW SW NW SW NE SW NE SE SW NW NW SW NE NW SE SE NE NW NW NE NE NW NW NW NE NE NW NW NE SE NE SW SW SE NE NE NE NW SE SE NW SW NW SE NE NW SW NW SE NW SE NE SW SE NW NW NE SE SW NE SE SW SE SE SE SE NE NE NE NW NW NE NW NE SE NW SE NW NE NE NE SW NE SW NW NE SE NE NW SW NE NE SE NE NE SE SE NW NE SE SE SW NE NW SE NW SE NE SE SE NW SW NE NW NW NW NE NE SW NW SW SE SE SW SW NE NW SW SW SE SE SW SE NE SW NW SW SW SE NW SW SW SW NW SW SE SW NE NE SE SW NE NE NW NE SE NW NE NW NE SW NW NE NW NW NE NW NE NE SW SW SE NW SE SE SW NW NE SE NE SE NW NE SW NW NE NW NE NE SE NE SW SW SE SW SE NE NE SE SW NW SW SE SE NW SW SW SE NW NE NW NE SW SW NW NE NW NE SW NW NW SW NW SW SW SW NW NW NE NE NE NW SE SW SW NE NE SW NW NE NW NW NW SW NW NE NE NE SE SE SW NE NW NE SW SE NE SW NE SE SW SE NW NE SW SW NE SE NW NW SW NW SE SW NW SW SE SW SE NW NW SE NE SW NW NE SE NW NW NE NW NE NE SE NE SE SW SW NW NE NE NE SW SW NE NW NE NW SE SE NW SE NE NW NE NE SE NW NE NW NW SE NE NE SE SW SE NW NW NE SE SE SW SW SW NW NE NW NW NE NE NW SE SW NE NW NE NW NE SE NE SE NE SW NW NW SE SE SW SE SW SW SE NE NW NE SE NW NE NE NW SW SW NW SW NW NW NE NE NW SW NW SE SW NW NE NW SE NW SE SW SW NE NE NE SE NE NW NE NE SE NW NW NE SW SE NE SE SE SW NE SE SW NW SW NE SW SE SW NW NW NW NW SW SE SE SE NW NE NE SE NE SW NW SE NE SW SW NW SW NW NE SE SW NW SW SE NW NE SW SE SE SW NE NE SE NW NE NW NE SW NE SE NE SW NW SE SE SW SW NW NE SE NW SE NE SE NE SW NW NW SE SW SE SE SE NE SW SE SE SE NW NW NE NE NW SE SE NE NE SE SE NW SE NE SW NW SE NW SW SE NW SE NE SW SE SE SW NE SE NE NW SW NW SW SE NW SE NW SW SE NW SE SE NE SE SE SW NW SE SE SE SW NW NE SW SE NE NE NE SE NW SW SW SE NW NW SW SW NE SE NW NE SW SE SW NW SE NW NE NW NE NE SE SE NW SW SW SW NE SE SW NW SW NW SW SE SE NE SW SW SW SW SE SW NE SW SE SE NW SE NE NW NE SW SE SE SE SE NW NE NE SE SW NE SE SW NE NE SW SW NW SW NE SE SE NW NW NW SW NE NE SE NE NW SW NW SW SW SE SE SE SE NE NW SE NE NE NE SW NW SE NW NE SE SW NW NE SW SW NE NE SE SW SW NW SE NW NE NW NE NW NW SE NE NE NW NE NW NE NW NW NE NE SW SW NE SW SE NW NW SW NE SE NE NW NE SW NW NE NW NE SE SW SW NE NE SE SE SW SW NW SW SE NE NE SE NE NW SE SW NW NE SE NE NW NW SW NW SW NE SW NE SE SW NW SW SE SE SW SW SW SW NW SE NW NE SW SW SW SE SW SE NW SE SW NW NW NW SE NW NE NE NE SW SE NW NW SW SW NE SE NE NW SE NW SE NE NE NW NW SE SE NE SW NE SE SE NE SE SW SW NW NW SE SW SW SW SW SW NW SE SW SE NE NE SE NW NW NE SE SW SE SE NE NW NW NE NE SW NW SE NE SE SW SW NW NW NE SW NW NW NE SW NW NE NW SW NW SE NE NE NW NE SE SE NW NE SE SW SE SW NW NE NW NE SW NW SW NE NE SE NW NE SW SW SE SW SW SE NW SW SE NW NW NE SE SW NE SW NW SW SE SE NW NW SE NW NW NW SW SW SE SE NW NW SW NW NW NW NE NW NW SW SE NE NW NW NE NE SE SW SW SW SW SE SW NE NE NW SW NW SW SW NE SE SE SE NE NW SW NW NE SW SE NE SW SW SW SE SE NE SW NW NW NE NW SW SE NE NW SW SW NW NE NE NW NE NE SE SW SE SE SW NW NE SE SW SW NW NE NE NW SE NW SE NE SE SE NW SW SW NW NW NW SE SE NW SW NW SW SE SE NE NE NE SE SE NW NE NE SE NE SE SE SE SW SE NW SE SW SE SE SE NE SW NW SW NE NE SE SW SE SE NE SW NW NE NE SE NE SE NW NE SW NW NE SE NW NW NE SE NE SE NW NE NW SW SW NW SE SW NW NW NE NE SW SW SW SW NE NW NE NW SW NW NW NE NE SE SE NE NW NE SE SW NE NW NE SE SE SW SW NW NE NW SW NW SE NE SW SE NE SE SW NE NE SW NE SE SW NE NW SW SW SW NW NE NW SW SE SE NW SE NW SW SE SW NW NE SW NW NE SE SW SE SW SE NE SE SE SW SW SE SE NW NW NE SE SE NW SW NW SE SW SW NW NE SW SE SW SW NW SE SW SW SW SW SW SW SE NE NW NE NW NW NE NW SW NE SW SW SW SE NW NE NW SW NE SE NE SE SE SW SW SW NW NW SW NE SE NE SW SE SW SW SE SE SE NE SE NW SW SW NW SW SW NE SW NW NE SE NE NW SW NE NE NW NE SW NW NE SE SW NW SW NW SE NE SE NE NW NE SW NE SE SE NE SW NE SE SW NW NW NE SW SW NE NW NE NW NW SW SE SW NW SE NW NW SE NE SW SE NW NE NW NE NW SE SW SE NE SE NW SE SW NW SE NW SW SW NW NE NW NW SW NE NE SW NE SE SW NW NE NE NW NW SE SE SE SW SW NW SE SE SW NW NE SE NE SW NW SW NW SE NE SW SE NW NW NE NE SW NW SW NW NE SW NW NE SW NW NE SE SW SE NE NW SE NW NE SE SE SW NE SW NE NE NW SE NE NE SE SW NW NW NE NW SW NE NW NE NE NE SE NW NE SE NW NE SE NE NW NE SE SE NW NE SE SW NW SW SW SE NW SE SW SW NE SE SE SW NW NE NE NE SE SE SE NW SE NW NW NW NE SW SW SW NE SE SE SE SW NE NW NW SE NW SW NW NW SW SW SW NW NW NW SE NE NE SW NW NE SE SE SW NW SE NW NE SE NE SW SE SE NW SE NE NW SW SW SW NW SW NW NW SE NW NE SE NW NE NW SW NW SW NW NE SE SW NW NE SE SE NE SE SW NW NE NW SE NW SE SE SW NE SW NE NW SE NW NE SW SE NW SW SW SW SE NW SW NW NE SE NW NE SE SE NE NE SE SE NE SW NE NE SE NE NW NW NW SW SE NE SE NE NW SW NW NE SW SE SE NW NE SW NW SE NW NW SW SE SE NE SW NE SE SW SW NE SE SE SW SW NE NE NW NW NW NE SW NW NW NW SW SW NE NE SW NE SE NE SW SW SW SE NE NW SE NW NE SE NW NW NW NW SW SE NW SE SE NW SE NW SW SW NE SE SW NE NE SW NE SE NW SW SW SW SE NE SW SE SE SW SW NE SE SW NW NW NE SW SW NE NE SE NW NW NE SW SW NW NE SE NE SE SW SE SW NW NW SW SW NW NW SE NE NW NE SE NW SE NE NW NE NE SW SE SE SW SE SW SE NW SW NE SE NW SE NW SW NW NE SE SE SW SE SW NW NE NW SW SW NW NE SE NE SW SW NW SW SE SW NW NW NE SE SE SW NW NE SW SW SW SW NE SE SW SW SE SW NE NE NE SW NW SW NW SE NW NE NE NW SW NE SW NW SW NE SE NW NW NW NE SE SW NE NE SW SW NE NE NE NE NW SE SE NW NE NW SW SE NE SE SE NE NW NE SE SW SW NW SE NE SW SE NE SW SE NE NW NW SE SE NE SE SW NE SE SW SE NW NE SW NW NW SW SE NW NW NE NW SW NW NW NW NW SE NW SW SE SW SE NW SW NW SW SW NW NW SW NW SE NW NE SE NW NE NE SE SE SW SE SW SE SW NE SW SW NE NW SW SW NE NE SE SW SE SE NE NW NE NE SW SW SE SW SW SE NW SW SW NE NW SE NE SE NE NW SE SE NE SW SW NW NE SW NW SW NW SW SW NE NE SW NW SE SW NE SE NW SW NW SW NW NW NW SE NE SE NE SE NW NE NE SW NW NE SW NW SE SW SW SW NW NE NE SE SW NW NE NE NE NE SE NE NE SE NE NE SW NW NW NE SW NE SW SE NE SE SE NE SW NE SW NE SE SE SW SE SE SW SW NE SE NW SW NW SE NW NW SW NW NE SW NW SW SE SE SE SW SW NW SE NW NE SE SE NW NE SE NE SE SE NW SW NE SW SW NE NE NE SW NE SE NW SE NW NW NE NE NE SE NE SW NE SE SE SW SW NE SE NW NE SE SW SE NE SW NE SW NW SE NW NW NE NE SE SE SE NE SE NE SW NW SW SE NE NW NE NW SE SE SW NW SE SE NW NE NW SW SW NE NE SE SW NW NE NW SW NW NW SW SW SE SE SE SW SW SW NW SE SW SE SE NW NW NE NE SW NW SW SE NE NE NE NW SE NE NE NE SW NE SE SW SW NE SE NW NE NW SE NW SW NE SW SW SW NE SW NE SW NW NE SE SE SE SE SW NE SE SE SE SE SE SW NE SW SE NW SW SW SE SE NW NE NE NW NE NE NW SW NE SW NE NW SE SW SE SE NW SW NW NW NE SW SE NE NW NE NW SW SW SW SE NE NW NE NW NW NE NE SW SE NW SE NE NW SW NW NW NE NE NW SE SE SW SW NW SW SE SW NW NE SE SW SW NW SE SE NE NW NE SE NW NE SW SW NW NE NW SW SW SE SE SW SE NW NW SW NW NE SW NW SW NW SW SE NW NE SW SE SW SE SE NE SW SE SW SE NW SW NE SE NW NE SE NW NE SW SE NE SW NE SW SE NE SW SE SW SE NW SE SW SE NE NW SE SE SW SE NE NW NW NE SE NE NW SW SW SW SW SE SE SW NW NW NW SW SE SE NE NW NW SE NW NW NE SW NW SW NW NE NE NE SW SE SW NE NW NW NE NE NE SE NE NW NE NE SW SW NE NW NW SE NW SW SE NW NW NW SE NE NE SW NW SW SE SE NW SE NE SW SW NE SE SW SW NW SW NW SW SW SE NE NW NE NE NW SW NW SW NW SW NE NW SE SW NW SE SE SW NW NE SE NE SE SE SW SE NE SW SW NW NW NE SE NW NE NE SW SW NW NW SW NW SW NW NW NE SW SW SE NW SW NW NE SW SE SE NE SE SW SW NW NE SW SW SW NW SW NW SW SE NE NE NW NE SW NE SE NW NE NW SW SW SW NW NE NW NW SW NE SW NE NW SE SW NW SE NW NE SE SE NW SE SW SW NE NE SE NE NW SW NW NW NE SW NE SE SW SW NW SE NW NE NW NW NE SE SE NW NW NE SE NE SW NW NE SE NW SW NW SE SW NW NE NE NW NE SE NE SW SW SW NW NE NE SW NE SE SE NE NW SW NW NW NE SE NE NW NE SE NW NW NW NE SW NE NE NW NE SW SW NW NW NE SW SE NE NE SW SW NW NW NW SE SE SE SE NW SE NE SW SE NW NE NW SE SE SW NE SW SE SE NE NW NE SE NW SE NW NE SE SW SW NW SW SW SE NW NE NW NW SE NE SW NE SE SW SE SE SW SW SE NW SE SE NW SE SE NW NE NW NW NE NE NW NW NE NE SW NE SW NE NW NW NW NE SE NE NE SE NE NE SW SW SW NE SW SW SW SE SW NE SE SE SW SE NE SW NW SW SW SW NW NE NE NW NE NE NE NW NE SE NW SW SW NW NW NW SE NE SW SW SE SE SE NE NE SW SW SW NW SW NE NE SE NW NW NE SE NW NW NW SW NE NW NE SW SW SE NE SE SE SW NW NE SE NE SE SW NE SE SW SW NW SE NE SE SW SE NW SW SE NE SW SW NE NE SE NW SW NW SW SW SW SE SW NW SW NE NE SW NW SE NW SE SE SW SE NW SW SW SW SW NW SE SE SW NW NW SW NW SW SW NW SE NW NW NE SE SW SE SE NE NE NW SE NW NW SW SE SE NE SE NW NE SW SE NE NW NW SW SE SE SE SE SW NW NW NE NE SE SE NW NW NE SE NW NW NW NE NW NE NE SE NE SW NE SW NE NW SE SW NW SW NE SE SW SW SW NE SW NW NE NW NW NW NE NW NW SE SW NW NE NE SW NW SE SW SE SE NE NE NW SE SW SE NW NE NE SE SE NE SE NW NW NW NW SE NE SE NE SE SW NW SW SW NE NE NW NW SE NE SE NE SW SW SW NW SW NW NE SW SE SW NW SE SW SW NW SE NW NE NE SE SW SE SE SW NE NW SW SE SW NE SE NW NE NW SE NW NW NE SW NE NW SE SE SW SW NE SE NW NW NW NE SW SE SE NE SE NE NE SW NW NE SW SW SE NE SW NW SW NW NE SE NW SW SW NE NW NE SE SE NW NE SW SW NW SE SW NW SE NW SE NE SW SE NW SW SE NW NW SW NE SW SW NE SE SW NE NW SW SW SW SE SW SE NW SW SE NE SE SE NE SW NE SE SW NE NE SE SW SE NE NE SW NE SE SE SW SW NW NW NE NE NW SW NE NE SW SE SE SE NW NE NW SW SW SW NW SW SE SE NW NW SW NW SE NE SE NE NW NW NE NW NE NE SE SE SW SW SE NW SE NE SW NW NE SW SE SE SW NW NE SE SW NW SE NE SE SW SE SW NW SW SE NE SE SW SE NW SW NW NW NE SE SE NW NW NE SW SE SE NW NW NW NW NE SE NW NE NE NW NW NE SW NE NE NE NW NE SW NW SE NW SW NW SW NW NW NW NE NE NE NW NE NW SW NW NE SW NE NW SW SW SW SW SE SW NW NE NW NE SW NE SW SW SW SW SW SW NE SE NW SE SW SW NE SE SE SW NE NW SE NW SW NW NE SW NW SE SE NW NE NE SW SW SE SE NW NE NW NE SW NW NW NE SE SW SW NE SE NW NE NW NE NW NW NE SE NW SW SW NE SE SE NW NE SW SW NW NE NW NE NW SE NW NW SE SW NE SW SE NE NW SW NW NE SE NE NW NE SE NW NE SE SW NE SE NE SW SE SW SW SW SW SE SE NW NW NE SW NW SE SW SW NW NW SE NW SE NW SE NW NW NE SE NE NW NE SE NE SE SW SE NE SE NE SW NW SE NE SW SW SW NW SW SE NE NE NW NE NE NW NE NW SW SE SW SE SW NW SE SE NW NE NW SE NW SW SW NE NE SW NE SE NW SE NW SE NW NE NE SW SE NW SW SE SE SW NW NW NE NE SE NW SE NE SW NE NW SW SW SW NE NW SE NE NW SE SE SW NW NE SW NW SW NW NE NW NW SW SE SE NW SE NE SW NE NW NE SW NW NE NW SW SW SE SE NW SE SW SW NE SE NW NE NW NE SE SE NW SE SW NW NW SW NW NW NW SE SE NW NW NW NE NW NW SE NE SE NW NW NW SW SE NE NE NE NE SW NW NE NE NW NW SW NW NE NW NE NW SW SE NE SE SW SE NW SE NE NE NW NW NE NW SE NE SW NW NW NW SW NW NE NE NW SW SW NW NE NE SW SE SE NW SW NW NE NW SE NE SE NE SW NW NE SW SW NW SW SE NW SE SW NE SE NW SE SW SW SW NE NW NW NW SW NE NE NE NW SW SE NW SE NW NE NW SW SW SE NW SE SW SE SE SW SE NW SW NW NW NE SE SW SE NW SE SW SW SE NE SE NW SW NW NE NE NW NE NE NE NE SW NW SE NW NE SE NE SE SE SW NW SW NE NW NE NE NW SW NE SE SW SW SW NW SE NE NW NE SE NE SW NW SW NW SE SW NE SW NE SE NW SE SE SW SW NE SW SW SW NW NE NW NW SW NW NW NE NW SE SE NE SW NW SE SE NW NW SE NE NW NE NW NE SE NE NW NE NE SW SW SW NW NE SW SW NE SE SE NW NW SW SE SW NW NW NE NE SE NE NE SW SE NE SW NW NE NW NE SW NW SW SE SW SW NW SE SE NE SE NW SE NW SW SE NE NW NW NW NE NW NE NE NW SW NW NE NW NE NE SW SE NW SW SW SW NE SW SW SE NW SW NE SE SE SW NW SW NW NW NE SE SE NW NE NW SW SE NE NW SW NW NE SW NW NE SW NW NW NE NW NW NE SW SE SW SW NE NE SE SE NW SW NE NW SW NW NW NW SE SW SE SE NE NW NW NE NW NE NE SE NE SW NW NE NE SW NE SE SW NW SE SE NE NW NE NE SW SW NE NE SE SW NW NW NE SW SW NE NW SW NE NW SE NW SW NE SE SE NW NE NW NE SE SE NE NW NE SW SE NE SE SE SW SE SE SW NW SW SW NE SW SE NE SE NE SE SW NW SW SE NW NE NE SW NE NE NW SW SE SW SE SE SW NW NE NW SE NW NW SW SE SE SE NW SE NW NE NE NW SW SW SW NW NW NE SE NE NW SW SE NW SW SW SE SW NW NE NE SE NW NE NW SW NE SW SE NW NE SW SW NW NW SE NW NW SW NE SW NE NW NE SE NE NE NW NE SE SE NE SE NE NW SW SE NE SE NE SE NW NW NW SE SE NE SE NE NE SE SE NW NE SE NE SW SE SE SW NE SE NE SW SE SW NE NE NE NE SE SE SE SW NE NW NW SW SW NE NW SE NE SE SW NW SW NW NE NW NE SW SE NE NW SW NW NE SW NE SW SE SW NE SE SW NE SE NW SW SW NE SW SE SW NE SE SE SE NE SE SW SE SE SE NE SE NE NE SE SW SW SE SE NW NE SE SW NE SE NW SE NE NE NE NE SW SW NE SE SW SW NE SW NW NE SW SW NE NE NE SW NW NE SW NE SW SW NW SE NE SE NW SE NE SW SE NE NE SW SW SE SW SE SW SE NE SE SE NW SE SE NW SW NW NE SE NE SW SW NW NE NE NE SW NW SE NW NW NE SE SW NW NE NE SE SW NW NE NW NE NW SE SW SE SW NE NE SE SW SE NW NE SW NE NW SW NW SW SW SE NE NW SW NE NW SW NW NE NW SE SW SW SW NW NE NE NE SE SW NW SE SW NW SE SW SW NW NW SW NE SE NE SE NE SW NE SW NE NW SE SW NE SE SW NW SW SE NW NE SW NW NW SE SE SW NE SE SE NE SW NW SE SW SW NE NW SW SW NW SE SE NE SE SE NE SW NE SW NW NE NW NE SW NW SE NE SW NW SE NW SE SW SE NE SE SE SW SE NE SW NW NE SW SE SW NE NE SE NE SE NE SW NE SE SE NW NW NW NE SE SW NW NE SE SE NE NW NW NE NW SW SE SW SE NE SW SW SE NW NE SE SE SW SE NE NW SE NE NW SW SW SW NE SE SW NE SE NE NE NW SW NE NE NE SE SE SE SW NE NW NE SW SE NW SW NW SW NW NE SE NE SW NE SE SW NE SW NW NE SW NW NW NE SE NE SE NW SW NW SE SE NW NE NW NE SW SE SE NE SW SW NE NW SW SW SE NE NE SW SW NW NE SE NE SW NE SE NW SW NE SE NW SE NW SW NW NE SE NE SW NW NE NW SE NW SW SW SW SW SE SW NW SE SW NE SW SE SW NE NE SW NW NE SE NW SW NE SE NW NW SW SE NE SW NE SE NE NW SW SW SW SE SW SE NE SE NW SE SE SW SE NW SW SE SW NE SW NE NE SW NE SE NE NW NE SE NW SE SW SW SE SE NE SW NE SE SW SE SE NE SE SW SW NW NW NW SW SE SE NE SW SW NW SE SE NW SW NW SW SW NW NW NE NE SW SE NW SW NW NW SW SE NW NE NE SW NE SE SE SE SE SW NW SW SE NE NE SE SE NW SE SW NE SE NE NE SE NE SE SE NE SW SW SW SE NW SE NW NE SE NE SE NE SE SW NW SW NW NE NW NE SE SW SW NW NE SW NE SE SW SW NW SE SW SW NW SW SW SE NW SW SE NW NE NE NW SW NW SE SE SW NW SE NE SW NW NW SE SW SE NW NE NE NW SW SW SW NW SW NW NE SW NE SW NW NE SE SE SE NW SW NW NW NE NE NW NE SE NE SW SW NW NE SE NE SW NW NE NW NE SW SW SW SE SW NE SE NW SW NE SE NW SW SE SE SW NW SE NE SE SW SW SE NW NW SW SW SW NW SE NW SW NW SE NE NE SW NE NW SE NW NW NW NW NW NE SW SW NW NE SW SW NW SE NW NW NW NE SE SW NW SW SW NW SE SE NE SW SW NW NE NE NE NE SW SE SE NE SW NW NW NE SE SW NW NE SW SE NE SW SE NW NE NE SE SE SW NW SE SW NW NE SE NW NE SE SE SW SW NW SE SW SE NE NW NE SE NW NE SW NE SE NE SW NW SE NW NW NE NE NW SE SW NE NE SW SW SW NW SW SE NW NW SW SW NW NE NW NW SW SE NW NE SE NW SW NE SW SW SW NW SE NW NE SE NW NW SW NW SW NW NE NE SE NE NE SE NE SW NE NE SW SE SE SW NW SW NE NW NE NW SE SE NW SW NE NW NE SW NE SE NE NE NW NE NW SW SW NW SW SE NE NW SW SE NW SW NW NE NW SE SE NW NE SE SW NW NE NE SE NE SE SW NW NE SE SW NW NE NW NE SE NE SE NE NE SW SE SW SW NE SE NW SE NW NW SE SW SE NE SW SW SW NE SE NW NE NW NW SW NW NE SE SE NW SE SW SE SE SW SE SE SW NE SE NW NW NE NW NW NE NE NE NW SW SW SE SW NW NW SE SE NW SE NE
